server/sonar-server/src/main/java/org/sonar/server/issue/ws/SearchResponseFormat.java

```diff
@@ -184,6 +184,7 @@ private void formatIssue(Issue.Builder issueBuilder, IssueDto dto, SearchRespons
       issueBuilder.setEffort(effortValue);
     }
     setNullable(dto.getLine(), issueBuilder::setLine);
+    setNullable(dto.getChecksum(), issueBuilder::setHash);
```

Comment:
I tested with squid:S1220, it seems to produce null line numbers, but empty, non-null line hashes. I suggest to not output empty hash attributes in those cases.